daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1604,19 +1605,19 @@ message DefTemplate {
   DefKey key = 10; // optional
 
   // Interfaces that this template implements.
-  repeated Implements implements = 13; // *Available in versions >= 1.dev*
+  repeated Implements implements = 13; // *Available in versions >= 1.15*
 }
 
 // Interface method definition.
-// *Available in versions >= 1.dev*
+// *Available in versions >= 1.15*
 message InterfaceMethod {
   Location location = 1;
   int32 method_interned_name = 2;
   Type type = 3;
 }
 
 // Interface definition.
-// *Available in versions >= 1.dev*
+// *Available in versions >= 1.15*
 message DefInterface {
```

Comment:
should we move `repeated TypeConName requires` to the end and add a comment `// *Available in versions >= 1.dev*`?

Reply:
Good catch.